packages/app-store/webex/lib/VideoApiAdapter.ts

```diff
@@ -0,0 +1,423 @@
+    },
+  };
+};
+
+const WebexVideoApiAdapter = (credential: CredentialPayload): VideoApiAdapter => {
+  //TODO implement translateEvent for recurring events
+  const translateEvent = (event: CalendarEvent) => {
+    //To convert the Cal's CalendarEvent type to a webex meeting type
+    console.log("event received in translateEvent", event);
+    /** @link https://developer.webex.com/docs/api/v1/meetings/create-a-meeting */
+    //Required params - title, start, end
+    //There are a ton of other options, what do we want to support?
+
+    console.log("calculated start", dayjs(event.startTime).utc().format());
+    return {
+      title: event.title,
+      start: dayjs(event.startTime).utc().format(),
+      end: dayjs(event.endTime).utc().format(),
+      recurrence: event.recurrence, //Follows RFC 2445 https://www.ietf.org/rfc/rfc2445.txt, TODO check if needs conversion
+      // timezone: event.organizer.timeZone, // Comment this out for now
+      agenda: event.description,
+      enableJoinBeforeHost: true, //this is true in zoom's api, do we need it here?
+      invitees: event.attendees.map((attendee) => ({
+        email: attendee.email,
+      })),
+      sendEmail: true,
+    };
+  };
+
+  const fetchWebexApi = async (endpoint: string, options?: RequestInit) => {
+    const auth = webexAuth(credential);
+    const accessToken = await auth.getToken();
+    console.log("result of accessToken in fetchWebexApi", accessToken);
+    console.log("createMeeting options in fetchWebexApi", options);
+    const response = await fetch(`https://webexapis.com/v1/${endpoint}`, {
+      method: "GET",
+      ...options,
+      headers: {
+        Authorization: "Bearer " + accessToken,
+        ...options?.headers,
+      },
+    });
+    const responseBody = await handleWebexResponse(response, credential.id);
+    return responseBody;
+  };
+
+  return {
+    getAvailability: async () => {
+      try {
+        const responseBody = await fetchWebexApi("meetings");
+
+        const data = webexMeetingsSchema.parse(responseBody);
+        return data.items.map((meeting) => ({
+          start: meeting.start,
+          end: meeting.end,
+        }));
+      } catch (err) {
+        console.error(err);
+
+        return [];
+      }
+    },
+    createMeeting: async (event: CalendarEvent): Promise<VideoCallData> => {
+      /** @link https://developer.webex.com/docs/api/v1/meetings/create-a-meeting */
+      try {
+        console.log("Creating meeting", event);
+        console.log("meting body", translateEvent(event));
+        console.log("request body in createMeeting", JSON.stringify(translateEvent(event)));
+        const response = await fetchWebexApi("meetings", {
+          method: "POST",
+          headers: {
+            "Content-Type": "application/json",
+          },
+          body: JSON.stringify(translateEvent(event)),
+        });
+        console.log("Webex create meeting response", response);
```

Comment:
https://github.com/calcom/cal.com/blob/main/packages/app-store/larkcalendar/api/events.ts#L52
Let's use logger for all console.logs